src/Uno.UI/UI/Xaml/WindowManagerInterop.wasm.cs

```diff
@@ -861,6 +861,11 @@ private struct WindowManagerDestroyViewParams
 		#region ResetStyle
 		internal static void ResetStyle(IntPtr htmlId, string[] names)
 		{
+			if (names == null || names.Length == 0)
+			{
+				// nothing to do
```

Comment:
```suggestion
				// nothing to do
				return;
```

Reply:
D'oh!  That's code I copied from elsewhere.